chain-signatures/crypto/Cargo.toml

```diff
@@ -14,4 +14,4 @@ sha3.workspace = true
 getrandom = { version = "0.2.12", features = ["custom"] }
 
 [dev-dependencies]
-web3.workspace = true
```

Comment:
Would it be hard to get rid of web3 completely? It's a heavy crate, and our Rust analyzer is already slow.